.github/workflows/ci.yml

```diff
@@ -0,0 +1,39 @@
+name: autofdo CI
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    types: [opened, reopened, synchronize]
```

Comment:
Does this trigger CI when user updates his PR? (Do we also want "edited"?)

Reply:
Assuming by update you mean adding more commits I think that is covered by `synchronize`. Edited covers the cases where title, description or base branch is `edited` [1]. Let me know if you still think it's worth adding the `edited`. 

[1] https://frontside.com/blog/2020-05-26-github-actions-pull_request/